api/src/convert/tx.rs

```diff
@@ -51,69 +50,144 @@ mod tests {
         for block_version in BlockVersion::iterator() {
             let alice = AccountKey::random(&mut rng);
             let bob = AccountKey::random(&mut rng);
-            let charlie = AccountKey::random(&mut rng);
 
-            let minted_outputs: Vec<TxOut> = {
-                // Mint an initial collection of outputs, including one belonging to
-                // `sender_account`.
-                let recipient_and_amounts: Vec<(PublicAddress, u64)> = vec![
-                    (alice.default_subaddress(), 65536),
-                    // Some outputs belonging to this account will be used as mix-ins.
-                    (charlie.default_subaddress(), 65536),
-                    (charlie.default_subaddress(), 65536),
-                ];
-
-                mc_transaction_core_test_utils::get_outputs(
-                    block_version,
-                    &recipient_and_amounts,
-                    &mut rng,
-                )
-            };
+            let fpr = MockFogResolver::default();
```

Comment:
nit: Can you use full words here to increase readability.? I'm not sure what this acronym stands for,

Reply:
yeah sure -- in a lot of test code right now `fpr` is `FogPubkeyResolver`, that's the trait that the `TransactionBuilder` requires when you want to make a new transaction

Reply:
i'm going to hold off on this in this PR because a lot of test code currently uses `fpr`. but we can look to improve that in a subsequent PR